apps/meteor/app/lib/server/functions/saveUserIdentity.ts

```diff
@@ -54,7 +57,7 @@ export async function saveUserIdentity({
 			return false;
 		}
 
-		if (!(await _setUsername(_id, username, user, updater))) {
+		if (!(await _setUsernameWithSession(session)(_id, username, user, updater))) {
```

Comment:
nitpick: the mix of ways of passing the transaction feels a bit odd. 
(feel free to ignore, it's more of a personal opinion)

Reply:
So, this is also one of the spots I used currying to avoid adding new parameters to the functions. Admitedly it's not the prettiest thing, but I did this mainly to functions that have other references to them, in an effort to minimize possible side effects or signature changes.